tests/Nox.Tests/DatabaseIntegrationTests/Models/Entities.TestEntityOneOrManyToExactlyOne.cs

```diff
@@ -0,0 +1,33 @@
+// Generated
+
+#nullable enable
+
+using Nox.Types;
+using Nox.Domain;
+using System;
+using System.Collections.Generic;
+
+namespace TestWebApp.Domain;
+
+/// <summary>
+/// .
+/// </summary>
+public partial class TestEntityOneOrManyToExactlyOne : AuditableEntityBase
+{
+    /// <summary>
+    ///  (Required).
+    /// </summary>
+    public Text Id { get; set; } = null!;
+
+    /// <summary>
+    ///  (Required).
+    /// </summary>
+    public Nox.Types.Text TextTestField2 { get; set; } = null!;
+
+    /// <summary>
+    /// TestEntityOneOrManyToExactlyOne Test entity relationship to TestEntityExactlyOneToOneOrMany OneOrMany TestEntityExactlyOneToOneOrManies
+    /// </summary>
+    public virtual List<TestEntityExactlyOneToOneOrMany> TestEntityExactlyOneToOneOrManies { get; set; } = new();
```

Comment:
Can we make testable relational entities generic to cut down on property definition.? The same is actual for "single" naviagation property

Reply:
no, because we generate entities fully from scratch the same way we do in production scenario